docs/dev-guide/canary-release-concepts.md

```diff
@@ -29,7 +29,7 @@ spec:
     app: payment-app
 ...
 
-apiVersion: apps/v1
+apiVersion: apps/v2
```

Comment:
Sorry should have been clearer. This should remain `v1` as it is a kubernetes api